smart-contracts/test/Lock/permissions/setKeyManager.js

```diff
@@ -0,0 +1,70 @@
+const { reverts } = require('truffle-assertions')
+const BigNumber = require('bignumber.js')
+// const { ZERO_ADDRESS } = require('hardlydifficult-ethereum-contracts')
```

Comment:
This is under `constants`

so `const {constants} = require('hd..
constants.ZERO_ADDRESS`

But inline is fine too

Reply:
May as well use constants, seeing as we have it available already. Will tweak this and push once more.